src/platforms/php/guides/symfony/config.mdx

```diff
@@ -7,17 +7,12 @@ In addition to the [configuration available in the PHP SDK](../configuration/opt
 
 All configuration for Symfony is done in `app/config/config.yml`.
```

Comment:
This is wrong, it's a remainder from Symfony 2.

If user uses Symfony Flex, it will start from the recipe: https://github.com/symfony/recipes-contrib/blob/master/sentry/sentry-symfony/3.0/config/packages/sentry.yaml
So it's in `config/packages/sentry.yaml

Reply:
Thanks, I've updated the references in this file.